src/server/scriptVersionCache.ts

```diff
@@ -684,7 +682,8 @@ namespace ts.server {
 
             // Skipped all children
             const { leaf } = this.lineNumberToInfo(this.lineCount(), 0);
-            return { oneBasedLine: this.lineCount(), zeroBasedColumn: leaf ? leaf.charCount() : 0, lineText: undefined };
+            if (!leaf) return { oneBasedLine: 1, zeroBasedColumn: 0, lineText: undefined };

```

Comment:
Why 1 and not 0?  (Asking out of total ignorance, never having seen this code before.)

Reply:
Since the line number is one-based (hence the `oneBased*` vs `zeroBased*` property names).

Reply:
Also, what's in the tree?  I'm guessing it might be a way to reuse data from previous versions of the file?

Reply:
It feels like things are already off the rails once `this.lineCount()` is zero, so maybe put a kickout to that effect at the top?  Kind of feels like it's supposed to behave the same as having no children, but maybe that would make `lineNumberAccumulator` zero and cause the same issue?

Reply:
(Looks like either you posted these comments again, or the GH UI is confused, so I'm resolving this...)